libraries/aws_ec2_traffic_mirror_filter.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AWSEc2TrafficMirrorFilter < AwsResourceBase
+  name 'aws_ec2_traffic_mirror_filter'
+  desc 'Audits EC2 Traffic Mirror Filter'
+
+  example "
+    describe aws_ec2_traffic_mirror_filter_rule(traffic_mirror_filter_id: 'test-traffic_mirror_filter') do
+      it { should exist }
+    end
+  "
+
+  def initialize(opts = {})
+    opts = { traffic_mirror_filter_id: opts } if opts.is_a?(String)
+    super(opts)
+    validate_parameters(required: %i(traffic_mirror_filter_id))
+    if opts[:traffic_mirror_filter_id] && !opts[:traffic_mirror_filter_id].empty? # Use instance_id, if provided
+      if !opts[:traffic_mirror_filter_id].is_a?(String) || opts[:traffic_mirror_filter_id] !~ /(^tmf-[0-9a-f]{8})|(^i-[0-9a-f]{17})$/
+        raise ArgumentError, "#{@__resource_name__}: `traffic_mirror_filter_id` must be a string in the format of 'lt-' followed by 8 or 17 hexadecimal characters."
+      end
+      @display_name = opts[:traffic_mirror_filter_id]
+      traffic_mirror_filters_arguments = { traffic_mirror_filter_ids: [opts[:traffic_mirror_filter_id]] }
```

Comment:
Change we change the line to 
traffic_mirror_filters_arguments = { traffic_mirror_filter_ids: [@display_name] }